tsconfig.json

```
@@ -17,6 +17,6 @@
     "allowJs": true,
     "types": ["node"]
   },
-  "include": ["src/*.ts", "src/**/*.ts"],
+  "include": ["src/*.ts", "src/**/*.ts", "e2e/server.ts"],
```

I actually prefer `server.ts` over `index.ts` but you seem to use `index.ts` elsewhere:
```suggestion
  "include": ["src/*.ts", "src/**/*.ts", "e2e/index.ts"],
```